tests/v2/test_lint.py

```diff
@@ -21,3 +21,19 @@ def test_check_experiments():
     tmp_problem = deepcopy(problem)
     tmp_problem["e1"].periods[0].time = tmp_problem["e1"].periods[1].time
     assert check.run(tmp_problem) is not None
+
+
+def test_check_incompatible_targets():
+    """Multiple conditions with overlapping targets cannot be applied
+    at the same time."""
+    problem = Problem()
+    problem.model = SbmlModel.from_antimony("p1 = 1; p2 = 2")
```

Comment:
:exploding_head:

Didn't know an SBML test model could be specified so simply.

Reply:
:muscle: